### PR TITLE
README.es.md

### PR DESCRIPTION
**Pull request by Yaten**
The Spanish readme file has been created as indicated in the pending tasks of the project.

Se ha creado el archivo readme en español según lo indicado en las tareas pendientes del proyecto

### CODE REVIEW

File: README.es.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+Puedes ver el desarrollo del bot en [https://twitch.tv/lucasmonstrocs](https://twitch.tv/lucasmonstrocs).
+Disfruta de nuestro Discord a través del enlace [https://discord.gg/HpvzwvNB](https://discord.gg/HpvzwvNB)
+
+# 📝 Licencia
+
+Derechos de autor © 2023 [**lucasmonstro**](https://github.com/lucasmonstro)  
+Este proyecto tiene licencia [MIT](https://opensource.org/licenses/MIT)
+
+
+
```
Comment (lucasmonstro) [inline README.es.md:127]: Remove essas duas linhas.

File: README.es.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+| Depósito de objetos apilables| :heavy_check_mark: |
+| Pescar                    | :x:                |
+| Comedor de alimentos      | :heavy_check_mark: |
+| Curación                  | :heavy_check_mark: |
+| Selección de objetivos inteligente| :heavy_check_mark: |
+| Recarga                   | :heavy_check_mark: |
+| Saqueo rápido             | :heavy_check_mark: |
+| Vender frascos            | :x:                |
+| Vender objetos            | :x:                |
+| Entrenar                  | :x:                |
```
Comment (lucasmonstro) [inline README.es.md:47]: Deixa isso bem formatado.

**Review by lucasmonstro — CHANGES_REQUESTED** [review]

**Comment by lucasmonstro** [comment]
@Yaten I will gonna ask for some spanish friend to check translate, is it ok for you?

**Comment by lucasmonstro** [comment]
@Yaten you need to remove TODOs lines from README.md and README.pt-BR files:

https://github.com/lucasmonstro/PyTibia/blob/main/README.md?plain=1#L95
https://github.com/lucasmonstro/PyTibia/blob/main/README.pt-BR.md?plain=1#L95

**Comment by lucasmonstro** [comment]
@Yaten Im using gitmoji to make commits, please, follow the same way and please, use english in commits. Thanks <3

**Comment by fmmendes** [comment]
> @Yaten I will gonna ask for some spanish friend to check translate, is it ok for you?

I can send to some people to review too, because there are several types of Spanish (Spain, Latin America, Argentina, Catalán, etc.)

**Pull request closed without merging**